app/routers/audio.py

```diff
@@ -16,7 +16,7 @@
     DEFAULT_SFX_VOL,
 )
 from app.dependencies import SOUNDS_PATH, get_db, templates
-from app.internal.security.dependancies import current_user
+from app.internal.security.dependencies import current_user
```

Comment:
Move to 3rd paragraph (git precommit hooks may solve this)

Reply:
I used the precommit hooks:)